de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/Activator.java

```diff
@@ -82,9 +81,8 @@ public String getAuthFilePathResolved() throws IOException {
 	}
 	
 	public String getAuthFilePath() {
-		// also provide this via cliManager
 		// this is an empty file by default
-		return authFilePath;
+		return ServerConfiguration.getAuthPropertiesFile();

```

Comment:
I think Activator doesn't need to know about individual properties. I suggest we create a generic method like `resolveBundlePath`, and use it whenever we want to get something from bundle resources